<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormContact</class>
 <widget class="QWidget" name="FormContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter Contact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Entreprise     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="EntrepriseEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mail                 </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MailEdit"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Telephone     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PhoneEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom                </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NomEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prenom          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PrenomEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PhotoButton">
       <property name="text">
        <string>Parcourir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="AnnulerButton">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="EnregistrerButton">
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
